pkg/crc/config/secret_config.go

```diff
@@ -56,7 +56,7 @@ func (c *SecretStorage) Unset(key string) error {
 }
 
 func NewEmptyInMemorySecretStorage() *SecretStorage {
-	keyring.MockInit()
+	// keyring.MockInit()
```

Comment:
This looks like a WIP commit? There's a CI failure because dbus-launch is missing, I guess it's related to this change?

Reply:
yeah, opened this PR to reproduce/demo the issue with the keychain not being accessible